src/main/resources/edu/wpi/grip/ui/pipeline/Step.fxml

```diff
@@ -1,53 +1,44 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<?import javafx.geometry.*?>
-<?import java.lang.*?>
-<?import java.net.*?>
 <?import javafx.scene.control.*?>
-<?import javafx.scene.image.*?>
-<?import javafx.scene.layout.*?>
-<?import javafx.scene.control.Separator?>
-<?import javafx.scene.control.TitledPane?>
-<?import javafx.scene.layout.AnchorPane?>
-<?import javafx.scene.layout.VBox?>
-<?import java.net.URL?>
-<?import javafx.scene.layout.HBox?>
-<?import javafx.scene.control.Label?>
-<?import javafx.scene.control.Button?>
-<?import javafx.scene.layout.BorderPane?>
-<?import javafx.scene.layout.StackPane?>
 <?import javafx.scene.image.ImageView?>
+<?import javafx.scene.layout.*?>
 
-<fx:root type="javafx.scene.layout.AnchorPane" xmlns="http://javafx.com/javafx/null"
-         xmlns:fx="http://javafx.com/fxml/1">
+<fx:root type="javafx.scene.layout.VBox" xmlns="http://javafx.com/javafx/null"
+         xmlns:fx="http://javafx.com/fxml/1" styleClass="step">
     <children>
-        <TitledPane fx:id="stepPane" alignment="CENTER" animated="false" collapsible="false" maxHeight="-Infinity"
-                    maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" styleClass="step">
+        <HBox>
+            <children>
+                <Button fx:id="deleteButton" onMouseClicked="#deleteStep" styleClass="delete" text="✕"
+                        HBox.hgrow="NEVER">
+                    <tooltip>
+                        <Tooltip text="Delete Step"/>
+                    </tooltip>
+                </Button>
+                <Pane HBox.hgrow="ALWAYS"/>
+                <Button fx:id="moveLeftButton" onMouseClicked="#moveStepLeft" styleClass="move-left"
```

Comment:
Is there a disabled property that this should have on it if it is at the left or right edge of the pipeline?